libcaf_core/caf/actor_system_config.hpp

```diff
@@ -252,6 +252,13 @@ class actor_system_config {
   size_t work_stealing_relaxed_steal_interval;
   size_t work_stealing_relaxed_sleep_duration_us;
 
+  // -- config parameters for the logger ---------------------------------------
+
+  std::string logger_filename;
+  std::string logger_verbosity;
```

Comment:
Can you give an example on how you see the switch working on an `atom_value`? The obvious one doesn't work:
```cpp
    switch (system_.config().logger_console) {
      case atom("NONE"):
...
```